source/interface/dav.js

```diff
@@ -19,7 +19,14 @@ function propsToStat(props, filename, isDetailed = false) {
         resourcetype: resourceType,
         getcontenttype: mimeType,
         getetag: etag
-    } = props;
+    } = props ? props : 
```

Comment:
Shouldn't props always be an object here? I don't think it should be optional.

Reply:
Props is somehow null in this case